edb/server/protocol/system_api.py

```diff
@@ -49,7 +49,7 @@ async def handle_request(
         if path_parts == ['status', 'ready'] and request.method == b'GET':
             if tenant is None:
                 # TODO(fantix): test the compiler pool
```

Comment:
Still TODO?

Reply:
Yeah, I haven't got to that part. I'd also like to reuse the syscon for pings if possible. I'll start a new PR for that.